Tests/Tests/DDSMocking.m

```diff
@@ -20,14 +20,15 @@ @implementation DDSMocking
 
 @implementation DDBasicMockArgument
 
-- (instancetype)initWithBlock:(void(^)(id object))block {
+- (instancetype)initWithBlock:(void (^)(id object))block {
```

Comment:
It doesn't format opening curly brackets in methods, does it?

Reply:
This change is the space after void. The configuration for function signature braces is to remove the new line. I have only modified the uncrustify configuration as I specified in the pull request so this change is based on the configuration that was already there but has not been applied in a while.

```
# Newline between function signature and open brace
nl_fdef_brace                           = remove        # string (add/force/ignore/remove)
```